eclipse/sw360
# fix(Release-UI): Vertical scrollbar for link release to project popup

Added scrollbar for link release to project popup. 

**Steps to test:**
1. Open a release and click on "Link to Project" button.
2. On the search project popup scroll bar should be present to scroll down and select project.

closes: #649
